functions.php

```diff
@@ -201,7 +201,7 @@ function twentysixteen_fonts_url() {
  * @since Twenty Sixteen 1.0
  */
 function twentysixteen_javascript_detection() {
-	echo "<script>(function(html){html.className = html.className.replace(/\bno-js\b/,'js')})(document.documentElement);</script>\n";
+	echo "<script>(function(html){html.className = html.className.replace(/\bno-js\b/,'') + ' js'})(document.documentElement);</script>\n";
```

Comment:
I assume the goal is to add the `js` class if it doesn't exist, but it doesn't feel right written this way.